tests/test_overdrive.py

```diff
@@ -323,12 +323,11 @@ def test_update_licensepool_error(self):
             "overdrive_availability_information.json"
         )
         api = DummyOverdriveAPI(self._db)
-        api.queue_response(response_code=500, content=availability)
+        api.queue_response(response_code=500, content="An error occured.")
```

Comment:
Our error handling only worked if you sent a non-200 response code but also provided the right data. That's the case we were testing but that's not actually how Overdrive deals with errors.